### PR TITLE
test plan

### PR DESCRIPTION
**Pull request by cakeben**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by timja** [comment]
nice 5 minutes saved by the looks of it

File: pipeline-templates/terraform-plan.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ jobs:
         displayName: Plan - ${{ parameters.zone }}
         inputs:
           command: 'plan'
-          commandOptions: '-out=${{ parameters.zone }}${{ parameters.build }}plan'
+          commandOptions: '-out=${{ parameters.zone }}${{ parameters.build }}plan  -parallelism=50'
```
Comment (cakeben) [inline pipeline-templates/terraform-plan.yaml:25]: @timja this saves me an additional 2minutes

Comment (timja) [reply pipeline-templates/terraform-plan.yaml:25]: Could be worth adding to private dns too?  Not sure what the build times are like thetr

Comment (cakeben) [reply pipeline-templates/terraform-plan.yaml:25]: I'm also thinking platform terraform, specifically front door changed if we add this to the apply

Comment (timja) [reply pipeline-templates/terraform-plan.yaml:25]: don't think it will help, there's not many resources there

**Pull request closed without merging**